crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -259,37 +259,48 @@ fn blueprint_ui(
             ui.add_space(ui.spacing().item_spacing.y);
 
             if let Some(space_view) = blueprint.viewport.space_view_mut(space_view_id) {
-                space_view.selection_ui(ctx, ui);
+                // TODO(jleibs): Is this the right place to create default state?
```

Comment:
as good as any I suppose 🤷 . Anything that needs more parameters can be handled on first frame where we have more information anyways (some stuff works already like this)